### ISSUE TITLE
озвончение основы _айт_ 

### ISSUE THREAD

**Issue by veramal** [tuffnatty/suddenly#223]
В словаре есть глагол _айтарға_ 'говорить' (№ 545) с основой _айт_-. На самом деле _т_ озвончается во всех встреченных примерах:
_Худаға килген хан **айдар** чооғы-чох сымна одырыбыстыр._ - Приехавший со сватовством хан молчком сидит, ему сказать нечего.
_Аны **айдып**, ибінзер айланып, нанып одыр._ - Это рассказав, домой возвращается, обратно идёт.
_Илееде тус ирт парды аның соонаң, аннаңар **айдарға** даа чарир ол киректеңер._ - Много времени прошло с тех пор, поэтому и рассказать можно об этом деле. 

Но сейчас вариант написания _айд_- перед гласной не анализируется.
В то же время глагол _артарға_ 'вешать' ведет себя по-другом - у него основа не озвончается. Вот из словаря: _чуған ниме артарға_ - развешать выстиранное [бельё], _пыроны кізее артыбызарға_ - свалить вину на другого. И так же не озвончаются и другие основы с сонантом и _т_ в конце: _иртерге_ 'проходить', пӱртерге 'подрубить': _ағастар пастарын пӱртерге_ подрезать верхушки деревьев; _кӧгенекті кӧп нимес пӱртібізерге_ укоротить немного платье, _сиртерге_ 'щелкать'. _сӱртерге_ 'мазать', _тартарға_ 'тянуть', _чортарға_ 'рысить', чыртарға 'рвать', хайтарға 'случиться'. Все прочие лексемы на сонант+_т_ - это каузативы.
Выходит, только _айт_ озвончается. Надо, вероятно, его как-то пометить в поле FORM и написать для него специальное морфонологическое правило.

**Comment by veramal** [comment]
Это можно поправить только после обновления словаря.

**Comment by veramal** [comment]
Сюда же: в кызыльском много примеров _Атар таң_ / _Адар таң_ "занимающаяся заря" (ат+Fut), в остальных значениях и диалектах вроде бы только _атар_.

**Comment by veramal** [comment]
И пат- может озвончаться: _падар ай_ "заходящая луна"

**Comment by adybo** [comment]
Это разве не к словарю вопрос?

  
>Среда, 16 апреля 2025, 14:46 +03:00 от veramal ***@***.***>:
> 
>И пат- может озвончаться:  падар ай "заходящая луна"
>—
>Reply to this email directly,  view it on GitHub , or  unsubscribe .
>You are receiving this because you are subscribed to this thread. Message ID:  <tuffnatty/suddenly/issues/223/2809325131 @ github . com>
>veramal left a comment  (tuffnatty/suddenly#223)
>И пат- может озвончаться:  падар ай "заходящая луна"
>—
>Reply to this email directly,  view it on GitHub , or  unsubscribe .
>You are receiving this because you are subscribed to this thread. Message ID:  <tuffnatty/suddenly/issues/223/2809325131 @ github . com> 
 
 
--
Anna Dybo
Отправлено из Почты  Mail

**Comment by veramal** [comment]
Отчасти к словарю, но новую словарную помету надо не только проставить, но еще и обработать. Пока это откладывается до обновления словаря (я сижу просматриваю опечатки и вношу правки в словарь пока).

**Comment by veramal** [comment]
Сейчас правило об основах на -т (I.1.2) записано так: "Корневая конечная т у некоторых основ (в основном глагольных) в литературном языке не озвончается: _атар (ат)_ ‘он будет стрелять’, _хатым (хат)_ ‘моя баба’. У подобных основ допустимы также и диалектные варианты с озвончением."
Очевидно, вторая часть её нереализована. При этом нужные основы имеют помету unvoiced (допишу в правило, чтобы не терялось). Но сделать допустимым звонкий вариант для них (ат, пат) - это одна задача, т.к. это позиция интервокала. У айт- же другой контекст. Приписала ему наоборот voiced, нужно по этой помете озвончать основу перед гласным.

**Comment by veramal** [comment]
Нет, это я что-то перемудрила. С _айт_- достаточно переписать инфинитив как _айдарға_, внесла поправку в словарь. Здесь остается разрешить "глухим" основам озвончаться.

**Comment by veramal** [comment]
_Айдар(га)_ не заработал. Видимо, ему нужно различать основы перед гласной и перед согласной, как для ис- и сис-. Тогда в alternat пишу айд-, в alternaten айт-.

**Comment by tuffnatty** [comment]
Добавил правило и вторую основу.

**Comment by veramal** [comment]
Айдар работает. Нужно еще разрешить озвончение для unvoiced.

**Comment by tuffnatty** [comment]
Разрешил.

**Comment by veramal** [comment]
_Адар, падар_ работают.
